staging/src/k8s.io/kubectl/pkg/cmd/cmd.go

```diff
@@ -440,6 +503,11 @@ func NewKubectlCommand(o KubectlOptions) *cobra.Command {
 	// Stop warning about normalization of flags. That makes it possible to
 	// add the klog flags later.
 	cmds.SetGlobalNormalizationFunc(cliflag.WordSepNormalizeFunc)
+
+	if os.Getenv(kubectlEnableAlphaCmdShadow) == "true" {
+		cmds.Flags().Bool("use-plugin", false, "If true, external kubectl plugin will be used instead built-in command. Only supports shadowing create command(e.g. kubectl create networkpolicy --use-plugin). This flag is alpha and may change in the future.")
```

Comment:
Although this flag will not be used(because it is only linked to real value when command is running and in our case it is not because we haven't decided yet to run built-in command or plugin), this is added to show the usage in `kubectl help`.